src/__stories__/loading-screen-story.tsx

```diff
@@ -0,0 +1,155 @@
+        return (
+            <ResponsiveLayout>
+                <Box paddingY={24}>
+                    <Stack space={16}>
+                        <Text2 regular as="p">
+                            Loading completed
+                        </Text2>
+                        <ButtonPrimary
+                            onPress={() => {
+                                location.reload();
+                            }}
+                        >
+                            Restart story
+                        </ButtonPrimary>
+                    </Stack>
+                </Box>
+            </ResponsiveLayout>
+        );
+    }
+
+    const textProps = multipleTexts
+        ? {
+              texts: [
+                  {title, description, time: textTime},
+                  {title: title + ' 2', description: description + ' 2', time: textTime},
+                  {title: title + ' 3', description: description + ' 3', time: textTime},
+              ],
+          }
+        : {title, description};
+
+    return (
+        <LoadingScreen
+            {...textProps}
+            isInverse={isInverse}
+            isLoading={isLoading}
+            onClose={() => {
+                setShowLoadingScreen(false);
+            }}
+        />
+    );
+};
+LoadingScreenStory.storyName = 'LoadingScreen';
+LoadingScreenStory.args = {
+    title: 'Title',
+    description: 'Description',
+    isInverse: false,
+    isLoading: true,
+    multipleTexts: false,
+    textTime: 5000,
+};
+LoadingScreenStory.argTypes = {
+    textTime: {if: {arg: 'multipleTexts'}},
+};
+
+type BrandLoadingSreenArgs = {
+    title: string;
+    description: string;
+    isLoading: boolean;
+    multipleTexts: boolean;
+    textTime: number;
+};
+
+export const BrandLoadingScreenStory: StoryComponent<BrandLoadingSreenArgs> = ({
+    title,
+    description,
+    multipleTexts,
+    textTime,
+    isLoading,
+}) => {
+    const [showLoadingScreen, setShowLoadingScreen] = React.useState(true);
+    if (!showLoadingScreen) {
+        return (
+            <ResponsiveLayout>
+                <Box paddingY={24}>
+                    <Stack space={16}>
+                        <Text2 regular as="p">
+                            Loading completed
+                        </Text2>
+                        <ButtonPrimary
+                            onPress={() => {
+                                location.reload();
+                            }}
+                        >
+                            Restart story
+                        </ButtonPrimary>
+                    </Stack>
+                </Box>
+            </ResponsiveLayout>
+        );
+    }
+    const textProps = multipleTexts
+        ? {
+              texts: [
+                  {title, description, time: textTime},
+                  {title: title + ' 2', description: description + ' 2', time: textTime},
+                  {title: title + ' 3', description: description + ' 3', time: textTime},
```

Comment:
`time` instead of `duration`?

`<FadeIn />` or `<Snackbar />` component use `duration`. I think is more consistent to use duration as prop name 🤔

Reply:
ok